teamlist.md

```diff
@@ -0,0 +1,64 @@
+Teams
+-----
+
+##### This is the team list as specified in cEP-0003
```

Comment:
I'd remove this line. Or add a normal text like, for more information about the teams, check out the cep